### PR TITLE
test(command-dev): replace cra test with eleventy test

### PR DESCRIPTION
**Pull request by erezrokah**
**- Summary**

Replaces the `create-react-app` site in our tests with an `eleventy` site.
In order to run the `cra` test we `npm ci` the site's dependencies. This takes about 25 seconds on my machine.
`npm ci` on the `eleventy` site takes about 4 seconds.

**- Test plan**

See test.

I had to `skip` one test due to https://github.com/netlify/cli/issues/1242 which I'll fix in a separate PR.

Screenshots on the Windows CI machine:
image
image

**- A picture of a cute animal (not mandatory but encouraged)**
image

### CODE REVIEW

File: tests/eleventy-site/_redirects [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/force.html         /test.html                      200!
-/something          /otherthing.html
+/force              /test                           200!
```
Comment (erezrokah) [inline tests/eleventy-site/_redirects:1]: `eleventy' puts files in their own directories (e.g. `test.html` is copied to `test/index.html`)

File: tests/eleventy.test.js [hunk 2]
```diff
@@ -34,67 +20,35 @@ test('homepage', async (t) => {
   const { url } = t.context.server
   const response = await got(`${url}/`).text()
 
-  t.true(response.includes('Web site created using create-react-app'))
-})
-
-test('static/js/bundle.js', async (t) => {
```
Comment (erezrokah) [inline tests/eleventy.test.js:40]: This is removed as it is no longer required

File: tests/eleventy.test.js [hunk 3]
```diff
@@ -34,67 +20,35 @@ test('homepage', async (t) => {
...
-
-  t.is(statusCode, 200)
-  t.true(body.length > BUNDLE_MIN_LENGTH)
-  t.true(headers['content-type'].startsWith('application/javascript'))
-  t.true(body.includes('webpackBootstrap'))
-})
-
-const BUNDLE_MIN_LENGTH = 1e2
-
-test('static file under public/', async (t) => {
```
Comment (erezrokah) [inline tests/eleventy.test.js:52]: Also removed as it is no longer required

File: tests/eleventy.test.js [hunk 4]
```diff
@@ -34,67 +20,35 @@ test('homepage', async (t) => {
...
 test('redirect test', async (t) => {
   const { url } = t.context.server
   const { body, statusCode, headers } = await got(`${url}/something`, { followRedirect: false })
 
   t.is(statusCode, 301)
-  t.is(headers.location, `/otherthing.html`)
-  t.is(body, 'Redirecting to /otherthing.html')
-})
-
-test('normal rewrite', async (t) => {
```
Comment (erezrokah) [inline tests/eleventy.test.js:70]: This one is skipped per https://github.com/netlify/cli/issues/1242

File: tests/eleventy.test.js [hunk 5]
```diff
@@ -34,67 +20,35 @@ test('homepage', async (t) => {
...
-  const body = await response.text()
-
-  t.is(response.status, 200)
-  t.true(response.headers.get('content-type').startsWith('text/html'))
-  t.true(body.includes('Web site created using create-react-app'))
+  t.is(headers.location, `/otherthing`)
+  t.is(body, 'Redirecting to /otherthing')
 })
 
-test('force rewrite', async (t) => {
```
Comment (erezrokah) [inline tests/eleventy.test.js:82]: This changed from getting `force.html` to `force`

File: tests/eleventy.test.js [hunk 6]
```diff
@@ -34,67 +20,35 @@ test('homepage', async (t) => {
...
-  const { body, statusCode, headers } = await got(`${url}/force.html`)
+  const { body, statusCode, headers } = await got(`${url}/doesnt-exist`)
 
   t.is(statusCode, 200)
   t.true(headers['content-type'].startsWith('text/html'))
-  t.true(body.includes('<h1>Test content</h1>'))
+  t.true(body.includes('Eleventy Site'))
 })
 
-test('robots.txt', async (t) => {
```
Comment (erezrokah) [inline tests/eleventy.test.js:91]: Removed as no longer required

**Review by ehmicky — APPROVED** [review]

**Pull request merged**